src/main/java/org/folio/innreach/controller/InnReachTransactionController.java

```diff
@@ -101,4 +102,13 @@ public InnReachResponseDTO handleMethodArgumentNotValidException(MethodArgumentN
     response.setErrors(innReachErrors);
     return response;
   }
+
+  @Override
+  @PostMapping("/inn-reach/d2ir/circ/recall/{trackingId}/{centralCode}")
+  public ResponseEntity<InnReachResponseDTO> recallItem(@PathVariable String trackingId,
+                                                        @PathVariable String centralCode,
+                                                        InnReachRecallItemDTO recallItem) {
+    var response = transactionService.recallItem(trackingId, centralCode, recallItem);
+    return ResponseEntity.ok(response);
+  }
```

Comment:
D2IR endpoints should be handled by `InnReachCirculationController`

Reply:
By the way, it's not clear to me what's the purpose of adding this method? It seems to be out of scope for the current story

Reply:
removed